README.md

```diff
@@ -242,8 +252,11 @@ TODO: when we have a prototype, link to/show an example of this in action.
 
 - `.show(options)`: shows the toast element,
 by toggling its `open=""` attribute to true.
+it will also start or reset the timeout of the toast,
+to show fo the provided `duration`,
+or a default `duration` of `2000`ms.
 The `options` include:
-    - `duration`: how long to show the toast, in milliseconds. Defaults to ???
+    - `duration`: how long to show the toast, in milliseconds. Defaults to `2000`.
```

Comment:
I think we should file an issue for the default timeout. It should include research on all the libraries, plus discussion about accessibility. E.g. some of the top Google search results for accessible toast suggest heuristics like 3 seconds + 60 milliseconds per character or similar.

Reply:
#37